Minio/DataModel/BucketOperationsReponse.cs

```diff
@@ -0,0 +1,71 @@
+/*
+ * MinIO .NET Library for Amazon S3 Compatible Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Minio.DataModel;
+using System.IO;
+using System.Net;
+using System.Xml.Serialization;
+
+namespace Minio
+{
+    public class BucketResponse: GenericResponse
+    {
+        internal string BucketName { get; set; }
+        public new BucketResponse Build()
+        {
+            return this;
+        }
+    }
+
+    public class VersioningConfig: BucketResponse
+    {
+        private VersioningConfiguration CurrentVersioningConfiguration { get; set; }
+        public string Status { get; private set;}
+        public bool MfaDelete { get; private set;}
+        public VersioningConfig(HttpStatusCode statusCode, string responseContent)
```

Comment:
```suggestion
        internal VersioningConfig(HttpStatusCode statusCode, string responseContent)
```

Reply:
Changed. Not calling it from outside anymore.